etc/compile/wait.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -Ee
```

Comment:
I think this flag makes it so that if any line returns non zero the script exits. Thats a little funny since you're handling the first command's return code explicitly, and the second one (on line 10) you probably don't want to return immediately (because you want to print the compile error on line 11).